tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/SortDirection.java

```diff
@@ -0,0 +1,13 @@
+package org.dcsa.conformance.standards.tnt.party;
+
+public enum SortDirection {
+  ASCENDING("ASC"),
+  DESCENDING("DESC");
```

Comment:
Is it possible to just use `ASC, DESC` instead of different name & value? Or are both really needed for the end user or in the standard? Just curious.